### PR TITLE
Add use case for adding person through json

### PR DESCRIPTION
**Pull request by aureliony**
Related to #129

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -328,44 +328,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-2a. System detects user input is invalid. <br>
-&emsp;2a1. System displays and error message. <br>
-&emsp;2a2. User enters new input. <br>
-&emsp;Steps 2a1-2a2 are repeated until user input is valid. <br>
-&emsp;Use case resumes from step 2.
+<div class="step">2a. AB3 detects user input is invalid.</div>
+<div class="sub-step">2a1. AB3 displays an error message.</div>
+<div class="sub-step">2a2. User enters new input.</div>
+<div class="sub-step">Steps 2a1-2a2 are repeated until user input is valid.</div>
+<div class="sub-step">Use case resumes from step 2.</div>
```
Comment (bryanyee33) [inline docs/DeveloperGuide.md:340]: Should this be `Use case resumes from step 3.` instead?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -328,44 +328,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-2a. System detects user input is invalid. <br>
-&emsp;2a1. System displays and error message. <br>
-&emsp;2a2. User enters new input. <br>
-&emsp;Steps 2a1-2a2 are repeated until user input is valid. <br>
-&emsp;Use case resumes from step 2. 
+<div class="step">2a. AB3 detects user input is invalid.</div>
+<div class="sub-step">2a1. AB3 displays an error message.</div>
+<div class="sub-step">2a2. User enters new input.</div>
+<div class="sub-step">Steps 2a1-2a2 are repeated until user input is valid.</div>
+<div class="sub-step">Use case resumes from step 2.</div>
```
Comment (bryanyee33) [inline docs/DeveloperGuide.md:365]: Should this be `Use case resumes from step 3.` instead?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -374,19 +371,18 @@ Use case ends.
...
-&emsp;3a2. User enters new input. <br>
-&emsp;Steps 3a1-3a2 are repeated until user input is valid. <br>
-&emsp;Use case resumes from step 3. <br>
+<div class="step">1a. AB3 displays no contacts.</div>
+<div class="sub-step">Use case ends.</div>
+<div class="step">3a. AB3 detects user input is invalid.</div>
+<div class="sub-step">3a1. AB3 displays an error message.</div>
+<div class="sub-step">3a2. User enters new input.</div>
+<div class="sub-step">Steps 3a1-3a2 are repeated until user input is valid.</div>
+<div class="sub-step">Use case resumes from step 3.</div>
```
Comment (bryanyee33) [inline docs/DeveloperGuide.md:385]: Should this be `Use case resumes from step 4.` instead?

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -395,17 +391,31 @@ Use case ends.
...
+4. AB3 deletes the contact.<br>
+   Use case ends.
+
+**Extensions**
+
+<div class="step">3a. AB3 detects user input is invalid.</div>
+<div class="sub-step">3a1. AB3 displays an error message.</div>
+<div class="sub-step">3a2. User enters new input.</div>
+<div class="sub-step">Steps 3a1-3a2 are repeated until user input is valid.</div>
+<div class="sub-step">Use case resumes from step 3.</div>
```
Comment (bryanyee33) [inline docs/DeveloperGuide.md:403]: Should this be `Use case resumes from step 4.` instead?

**Review by bryanyee33 — COMMENTED** [review]
LGTM, but the use case resume steps are incorrect.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (aureliony) [reply docs/DeveloperGuide.md:340]: Fixed

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (aureliony) [reply docs/DeveloperGuide.md:365]: Fixed

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (aureliony) [reply docs/DeveloperGuide.md:385]: Fixed

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (aureliony) [reply docs/DeveloperGuide.md]: Fixed

**Review by bryanyee33 — APPROVED** [review]
LGTM.

**Pull request merged**